### DISCUSSION TITLE
How to Use Typeinfo to store the typeid of a json object value

### DISCUSSION THREAD

**Discussion by beemibrahim** [nlohmann/json#3253] [Q&A] [answered]
How To Use Type Info to check what type a object is .

**Comment by nlohmann** [comment]
What do you mean with "type info" and "object"?

Maybe https://json.nlohmann.me/api/basic_json/type_name/ and https://json.nlohmann.me/api/basic_json/type/ can be helpful.

**Comment by beemibrahim** [comment]
i want to find the type of a json obect value and see if it is an std :: string

**Reply by nlohmann** [reply]
See https://json.nlohmann.me/api/basic_json/is_string/

**Reply by beemibrahim** [reply]
okay that works

**Comment by beemibrahim** [comment, marked as answer]
See https://json.nlohmann.me/api/basic_json/is_string/
this is the right answer
